src/main/java/seedu/address/logic/commands/DeleteTimeCommand.java

```diff
@@ -31,11 +31,14 @@ public class DeleteTimeCommand extends Command {
 
     public static final String COMMAND_WORD = "deleteTime";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes a free time to the person identified "
-            + "by the index number used in the displayed person list. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes a free time from the person identified "
+            + "by the index number used in the displayed person list. \n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + PREFIX_FREETIMETAG + "FREE TIME TAG...\n"
+            + "Example: " + COMMAND_WORD + " 1 "
             + PREFIX_FREETIMETAG + "Mon:1300-1400";
```

Comment:
Added more detail to the error message, where it is incomplete (gramatically wrong) previously.